config/prow/cluster/monitoring/prow_alertmanagerconfig.yaml

```diff
@@ -0,0 +1,41 @@
+# PROW_INSTANCE_SPECIFIC
+# Slack endpoint, or even different methods of alerting
+apiVersion: monitoring.coreos.com/v1alpha1
+kind: AlertmanagerConfig
+metadata:
+  name: alertmanager-prow
```

Comment:
Based on the file name this should be "prow". Again, I'm not sure if this actually matters.

Reply:
afaict the first part of the file name is pretty much used for naming the generated bazel targets as https://github.com/kubernetes/test-infra/blob/985ef5e7217e4bc0d8abbe01e7e4d2ab439bd43e/prow/def.bzl#L158 and https://github.com/bazelbuild/rules_k8s/blob/f1c6399cdd691b7aca90073398e8f690ec8992c6/k8s/object.bzl#L481. Will unhold tomorrow